specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -0,0 +1,3448 @@
+                  "schema": {
+                    "$ref": "#/definitions/TransactionsListResult"
+                  }
+                },
+                "default": {
+                  "description": "Error response describing why the operation failed.",
+                  "schema": {
+                    "$ref": "#/definitions/ErrorResponse"
+                  }
+                }
+              }
+            }
+          },
+          "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/policies/default": {
+            "get": {
+              "tags": [
+                "Policy"
+              ],
+              "operationId": "Policy_GetByBillingProfile",
+              "description": "The policy for a given billingAccountId and billingProfileId.",
+              "externalDocs": {
+                "url": "https://docs.microsoft.com/en-us/rest/api/consumption/"
+              },
+              "x-ms-examples": {
+                "PolicyByBillingProfile": {
+                  "$ref": "./examples/Policy.json"
+                }
+              },
+              "parameters": [
+                {
+                  "$ref": "#/parameters/billingAccountIdParameter"
+                },
+                {
+                  "$ref": "#/parameters/billingProfileIdParameter"
+                },
+                {
+                  "$ref": "#/parameters/apiVersionParameter"
+                }
+              ],
+              "responses": {
+                "200": {
+                  "description": "OK. The request has succeeded.",
+                  "schema": {
+                    "$ref": "#/definitions/Policy"
+                  }
+                },
+                "default": {
+                  "description": "Error response describing why the operation failed.",
+                  "schema": {
+                    "$ref": "#/definitions/ErrorResponse"
+                  }
+                }
+              },
+              "x-ms-pageable": {
+                "nextLinkName": "nextLink"
+              }
+            },
+            "put": {
+              "tags": [
+                "Policy"
+              ],
+              "operationId": "Policy_Update",
+              "description": "The operation to update a policy.",
+              "x-ms-examples": {
+                "UpdateBillingProfile": {
+                  "$ref": "./examples/UpdatePolicy.json"
+                }
+              },
+              "parameters": [
+                {
+                  "$ref": "#/parameters/apiVersionParameter"
+                },
+                {
+                  "$ref": "#/parameters/billingAccountIdParameter"
+                },
+                {
+                  "$ref": "#/parameters/billingProfileIdParameter"
+                },
+                {
+                  "name": "parameters",
+                  "in": "body",
+                  "required": true,
+                  "schema": {
+           
```

Comment:
Recommend using a string enum here (with values of "true" and "false") as that is more flexible to extend in the future